plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/aps/system/services/content/model/attribute/AbstractResourceAttribute.java

```diff
@@ -272,14 +272,16 @@ protected void addResourceMetadatasElement(Element attributeElement) {
         }
         Element metadatasElement = new Element("metadatas");
         this.getMetadatas().keySet().stream().forEach(key -> {
-            Map<String, String> map = this.getMetadatas().get(key);
-            map.keySet().stream().forEach(langCode -> {
-                Element metadataElement = new Element("metadata");
-                metadataElement.setAttribute("key", key);
-                metadataElement.setAttribute("lang", langCode);
-                metadataElement.setText(map.get(langCode).trim());
-                metadatasElement.addContent(metadataElement);
-            });
+            if (null != key) {
```

Comment:
I don't think this is necessary. A key in a map can't be null. If there was an exception here it could be that getMetadatas() was null?